### ISSUE TITLE
Can't connect to local unleash server

### ISSUE THREAD

**Issue by romulorosa** [Unleash/unleash-client-python#164]
Hi there!

**Describe the bug**
I am running Unleash locally using the docker image (v 4.0.5) and the python client (UnleashClient==4.4.0) to play a bit with it. Turns out that the client insist on not connecting to my local server.
I created an API key in the admin section and tried to run my client accordingly to what is described here https://docs.getunleash.io/sdks/python_sdk.

The documentation and the information inside the app is misleading because it is not clear if it is necessary to append `/api` or not. For instance, https://github.com/Unleash/unleash-client-python/blob/master/README.md omits the `/api/` but in the other docs you can find it.

![Screen Shot 2021-06-29 at 14 42 32](https://user-images.githubusercontent.com/15314978/123801217-5c743a80-d8ea-11eb-9c0e-07182af91d98.png)

**To Reproduce**
Anyway, I tried both urls and the funny thing is, both fail but with different errors. 

1. Configure API Access in http://localhost:4242/admin/api
2. Follow https://docs.getunleash.io/sdks/python_sdk

Sample code:
### 1 - With /api
```
In [5]: from UnleashClient import UnleashClient

In [6]: client = UnleashClient(url="http://localhost:4242/api/", app_name="content", custom_headers={'Authorization': "c795214e5ccf71a83238ba5a496a440b7e68379074172d9750bed5bb582c81d6"})

In [7]: client.initialize_client()
```
Stack trace
```
Unleash Client registration failed due to unexpected HTTP status code.
Unleash Client feature fetch failed due to unexpected HTTP status code.
Unleash Client feature fetch failed due to exception: Unleash Client feature fetch failed!
Traceback (most recent call last):
  File "/env/lib/python3.8/site-packages/UnleashClient/api/features.py", line 50, in get_feature_toggles
    raise Exception("Unleash Client feature fetch failed!")
Exception: Unleash Client feature fetch failed!
Unable to get feature flag toggles, using cached provisioning.
Cache Exception: '/client/features'
Unleash client does not have cached features. Please make sure client can communicate with Unleash server!
```
### 2 - Without /api
```
In [1]: from UnleashClient import UnleashClient

In [2]: client = UnleashClient(url="http://localhost:4242/", app_name="content", custom_headers={'Authorization': "c795
   ...: 214e5ccf71a83238ba5a496a440b7e68379074172d9750bed5bb582c81d6"})

In [3]: client.initialize_client()
Unleash Client registration failed due to unexpected HTTP status code.
Unleash Client feature fetch failed due to exception: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/env/lib/python3.8/site-packages/UnleashClient/api/features.py", line 52, in get_feature_toggles
    return resp.json()
  File "/env/lib/python3.8/site-packages/requests/models.py", line 898, in json
    return complexjson.loads(self.text, **kwargs)
  File "/env/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/env/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "env/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Unable to get feature flag toggles, using cached provisioning.
Cache Exception: '/client/features'
Unleash client does not have cached features. Please make sure client can communicate with Unleash server!
```
This second scenario shows that the connectivity is not a problem, because the client receives a 200 from my local server, however the response seems to be unexpected. The content is a HTML

**Expected behavior**
Connect to local server. Not sure if the problem is the URL or the headers. I could not find any other examples for this scenario.

**Comment by ivarconr** [comment]
Hi there. 

The url should include the /api path so the correct one is http://localhost:4242/api

I suspect it is the wrong API token. You can verify that my enabling more logging or simply do a curl against http://localhost:4242/api/client/features with the API token set as the Authorization header.

**Comment by romulorosa** [comment]
Hi @ivarconr,

Thank you. After spending some time debugging I found the problem. The problem is the url and the slash `/` should not be included after `/api` (as suggested by the print that I attached previously) as you pointed out. 
I was checking the source code and perhaps the url strip is wrong (https://github.com/Unleash/unleash-client-python/blob/master/UnleashClient/__init__.py#L55). I believe the correct approach would be `url.rstrip('/')`
